File: backend/entity/payment.go
package entity

import "gorm.io/gorm"

type Payment struct {
    gorm.Model

    WorkID       uint    `json:"work_id"`
    Work         Work    `gorm:"foreignKey:WorkID;references:ID"`

    BookerUserID uint    `json:"booker_user_id"`
    BookerUser   Users   `gorm:"foreignKey:BookerUserID;references:ID"`

    PosterUserID uint    `json:"poster_user_id"`
    PosterUser   Users   `gorm:"foreignKey:PosterUserID;references:ID"`

    Wages        float64 `json:"wages"` // เปลี่ยนให้ตรงกัน
}

File: frontend/src/interfaces/Payment.ts
export interface PaymentInterface {
    id: number; // ID ของการชำระเงิน
    wages: number; // จำนวนเงินที่ชำระ
    booker_user_id: number; // ผู้ที่ให้
    poster_user_id: number; // ผู้ที่ถูกให้
    created_at: string; // วันที่สร้างการชำระเงิน
    work:  number;
}